### DISCUSSION TITLE
Ideas and New Improvement Features

### DISCUSSION THREAD

**Discussion by Sid72020123** [Sid72020123/scratchconnect#15] [Ideas]
If you have any ideas to improve this library then discuss here! We will surely try to add them!

**Comment by Quantum-Codes** [comment]
"We" means you and______?

**Reply by Sid72020123** [reply]
LOL The Contributors! Like you and me. You can help me  if you want to :) or else no.
Its ok :)

**Reply by Quantum-Codes** [reply]
Lol ok
